oauth2_http/pom.xml

```diff
@@ -90,13 +89,6 @@
           </excludes>
           <reportNameSuffix>sponge_log</reportNameSuffix>
         </configuration>
-        <dependencies>
```

Comment:
Is it really needed? It blocks running JUnit5 tests. Only old tests run with the dependency.